app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ActiveRecord::Base
+  has_many :surveys
+  has_many :questions
+  has_many :choices
+  has_many :answers
+  validates :username, presence: true
+  has_secure_password
+end
```

Comment:
In general, I expected to see more instance methods in your models. I'm guessing quite a lot of the logic for your app is currently in the controllers. Think about what work is being done in the controllers, see if you can "name that work" and then decide "whose job it is".